OJP/OJP_Trips.xsd

```diff
@@ -228,31 +223,71 @@
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="minChanges">
+				<xs:annotation>
+					<xs:documentation>Minimise the number of interchanges as the first criterion. Expected strategy.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leastWalking">
+				<xs:annotation>
+					<xs:documentation>Shortest walking distance in meters, summed over all legs.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leastCost">
+				<xs:annotation>
+					<xs:documentation>Cheapest fare, taking into account the applicable reductions. Might not base on actual cost, but an estimation. Expected strategy.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 			<xs:enumeration value="leastDistance">
 				<xs:annotation>
-					<xs:documentation>Least distance in meter. Mostly used for ALTERNATIVE MODE OF OPERATION</xs:documentation>
+					<xs:documentation>Least distance in metres. Mostly used for ALTERNATIVE MODE OF OPERATION and for ItModesToCover.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="earliestArrival">
+				<xs:annotation>
+					<xs:documentation>Earliest possible arrival time respecting the time constraints. (forward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="latestDeparture">
+				<xs:annotation>
+					<xs:documentation>Latest departure time for a given arrival time. (backward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="earliestArrivalAndLatestDeparture">
+				<xs:annotation>
+					<xs:documentation>Combines earliestArrival and latestDeparture, allowing to compress the departure time. (forward-backward-forward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="minNonLevelEntrances">
+				<xs:annotation>
+					<xs:documentation>The user wants to minimize non level entrances on the trip. this is useful for PRM who still can use some non-level entrances.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="minStairs">
+				<xs:annotation>
+					<xs:documentation>The user wants to minimize stairs and steps on the trip. This is useful for PRM who still can use some steps/stairs.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="bestForVisualImpairment">
+				<xs:annotation>
+					<xs:documentation>The user wants to avoid transfers without tactile guidance, as well as platforms and vehicles without auditory signals.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="bestForAudioImpairment">
+				<xs:annotation>
+					<xs:documentation>The user wants to avoid transfers without guidance for people with audio impairment, as well as platforms and vehicles without visual signals.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>The user wants to avoid transfers without guidance for people with auditory impairment, as well as platforms and vehicles without visual signs.</xs:documentation>
```